tests/unit_tests/c_wrappers/integration/test_field_parameter.py

```diff
@@ -0,0 +1,589 @@
+import math
+import os
+import stat
+from argparse import ArgumentParser
+from pathlib import Path
+from textwrap import dedent
+
+import numpy as np
+import numpy.testing
+import pytest
+import xtgeo
+
+from ert.__main__ import ert_parser
+from ert._c_wrappers.enkf import EnKFMain, ResConfig
+from ert.cli import ENSEMBLE_SMOOTHER_MODE
+from ert.cli.main import run_cli
+from ert.libres_facade import LibresFacade
+
+
+def create_runpath(config, active_mask=None):
+    active_mask = [True] if active_mask is None else active_mask
+    res_config = ResConfig(config)
+    ert = EnKFMain(res_config)
+    prior = ert.load_ensemble_context("default", active_mask, 0)
+    ert.sample_prior(prior.sim_fs, prior.active_realizations)
+    ert.createRunPath(prior)
+    return ert
+
+
+def load_from_forward_model(ert):
+    facade = LibresFacade(ert)
+    realizations = [True] * facade.get_ensemble_size()
+    return facade.load_from_forward_model("default", realizations, 0)
+
+
+def write_grid_property(name, grid, filename, file_format, shape, buffer):
+    arr = np.ndarray(shape=shape, buffer=buffer, dtype=buffer.dtype)
+    prop = xtgeo.GridProperty(
+        ncol=shape[0], nrow=shape[1], nlay=shape[2], values=arr, grid=grid, name=name
+    )
+    prop.to_file(filename, fformat=file_format)
+    return arr
+
+
+def test_unknown_file_extension(tmpdir):
+    with tmpdir.as_cwd():
+        config = dedent(
+            """
+        JOBNAME my_name%d
```

Comment:
Is specifying JOBNAME neccessary here?

Reply:
It seems not, ill it from all tests in this file